src/Tabs/tabStyles.ts

```diff
@@ -3,17 +3,15 @@ import styled from '@emotion/styled';
 export const TabBar = styled.div<any>(({ theme }) => ({
     display: 'flex',
     flexFlow: 'row nowrap',
-    backgroundColor: theme?.colors?.white
+    backgroundColor: theme?.colors?.white,
+    marginLeft: '-2.5rem'
 }));
 
 
 export const TabStyle = styled.span<any>(({ theme, isSelected }) => ({
-    margin: '2.5rem',
     color: theme?.colors?.pavement,
-    padding: '0.5rem 0 0.625 0',
-    '& + .tab': {
-        marginLeft: '2.5rem'
-    },
+    padding: '0.5rem 0 0.625rem 0',
+    marginLeft: '2.5rem',
```

Comment:
On both of the pages I checked just now, the tabs are supposed to be flush against the left side of the container.  I think that `marginLeft: 2.5rem` rather than `& + .tab { marginLeft: 2.5rem }` will cause problems with this design... am I missing a reason to make the margin unconditional?

Reply:
So, unfortunately '& + .tab' wouldn't work in the new Styled Components as we dont have tab class now and the unique class name is created by the styled components which we cannot use. To solve this I have added MarginLeft which would set margin:2.5rem to all the tabs inside the container but since we dont need marginLeft on the first tab inside the container I have given "marginLeft:-2.5rem" to the container which is hack that would not set any marginLeft on the first child. I have tested the page with these changes and it is rendering fine the way it used to render before.

Reply:
Also added the screenshot of the lakehouse after integrating the lakefront changes manually tto seee how it would render. Please have a look and let me know if you have any questions.

Reply:
Ah, OK -- the `-2.5rem` to the container isn't part of this PR, so I didn't realize that was part of the fix.  It looks like styled components do support adjacent sibling selectors...?  Take a look at https://github.com/styled-components/styled-components/issues/74#issuecomment-296757178 -- I think if this works, it would be preferable to imposing constraints on the container.

Reply:
@NehalBhandariSlalom, Rocky is correct. If you need help with this method of styling, let me know.

Reply:
Added Sibling selector for Tabs. Thanks for suggestion.

Reply:
Following up from a separate discussion: it looks like the reason `& + &` doesn't work is because the selected tab gets a different generated style object (and therefore a different generated class) than the non-selected tab.  I'm OK with this workaround.